src/metaswitch/clearwater/cluster_manager/main.py

```diff
@@ -10,42 +10,49 @@
 """Clearwater Cluster Manager
 
 Usage:
-  main.py --mgmt-local-ip=IP --sig-local-ip=IP --local-site=NAME --remote-site=NAME --remote-cassandra-seeds=IPs --uuid=UUID --etcd-key=KEY --etcd-cluster-key=CLUSTER_KEY
-          [--signaling-namespace=NAME] [--foreground] [--log-level=LVL]
-          [--log-directory=DIR] [--pidfile=FILE] [--cluster-manager-enabled=Y/N]
+  main.py --mgmt-local-ip=IP --sig-local-ip=IP --local-site=NAME --uuid=UUID (--etcd-key=KEY | --db-key=KEY) (--etcd-cluster-key=CLUSTER_KEY | --db-cluster-key=CLUSTER_KEY)
+          [--remote-site=NAME] [--remote-cassandra-seeds=IPs] [--cassandra-container-id=ID] [--signaling-namespace=NAME] [--foreground] [--log-level=LVL]
+          [--log-directory=DIR] [--pidfile=FILE] [--cluster-manager-enabled=Y/N] [--etcd | --consul]
 
 Options:
   -h --help                      Show this screen.
-  --mgmt-local-ip=IP             Management IP address
-  --sig-local-ip=IP              Signaling IP address
+  --mgmt-local-ip=IP             Management IP address (for the back-end)
```

Comment:
what does "the back-end" mean?  Consul, or Cassandra?  If the former, is it "local"?